public/static/js/task_admin.js

```diff
@@ -38,6 +38,15 @@ mciModule.controller('AdminOptionsCtrl', ['$scope', '$window', '$rootScope', 'mc
         }
     }
 
+    function doModalFailure(message, data, reload){
+        $scope.closeAdminModal();
+        if (reload) {
```

Comment:
I don't think there's any need to reload on error. It's reloaded on success to update displays

Reply:
You're right- I've cleaned up the function so that it doesn't include the reload logic anymore.